template/hooks/common/functions.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Common functions for cloud hooks.
+
+status=0
+
+env_refresh() {
+
+  echo "Running env_refresh() for environment: $target_env"
+
+  #create drush command
+  drush_cmd="drush @$site.$target_env"
+  echo $drush_cmd
+
+  # Prep for BLT commands.
+  repo_root="/var/www/html/$site.$target_env"
+  export PATH=$repo_root/vendor/bin:$PATH
+  cd $repo_root
+
+  echo "Update starting..."
+  blt deploy:update
```

Comment:
(I'm working with @greylabel on a platform build and we're making use of BLT.)

I see why your example was working: you manually CD to the 'repo root' in the script (in quotes since it's not a proper repo), which means vendor/bin/blt is able to find vendor/acquia/blt/blt.sh, and therefore doesn't fail. That works, but it would be nice to minimally figure out how to make vendor/bin/blt invokable from anywhere in the build.